CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 0.4.0
+
+* Add an extension method on String to allow easy creation of Characters from
```

Comment:
Use "\`String\`" and "\`Characters\`" to show source names as code.

-> "... to allow easy access to the `Characters` of the string."